mrp_bom_tracking/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module adds track visibility to some fields of mrp boms.
```

Comment:
Can you elaborate a bit more, it is not just that, it log in the chatter any change in the bom lines too.